lib/attrsets.nix

```
@@ -302,6 +302,35 @@ rec {
         in mapAttrs g set;
     in recurse [] set;
 
+  /* Builds a map from <attr>=value to <attr>.<item>=value for each item in
+   * the list.
+   *
+   * This is mainly useful for flake-like structures.
+   *
+   * Type:
+   *   mapAttrsJump ::
+   *     [String] -> (String -> AttrSet) -> AttrSet<String, AttrSet>
```

Same with the type here

```suggestion
   *     [a] -> (a -> AttrSet<b, c>) -> AttrSet<b, AttrSet<a, c>>
```

---

From the type, I would call it something like `genAttrsTransposed`.

---

Good idea